examples/usercommands.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+from flask import Flask
+
+# This is just here for the sake of examples testing
+# to make sure that the imports work
+# (you don't actually need it in your code)
+sys.path.insert(1, ".")
+
+from flask_discord_interactions import DiscordInteractions  # noqa: E402
+from flask_discord_interactions.context import ApplicationCommandType
+
+
+app = Flask(__name__)
+discord = DiscordInteractions(app)
+
+# Find these in your Discord Developer Portal, store them as environment vars
+app.config["DISCORD_CLIENT_ID"] = os.environ["DISCORD_CLIENT_ID"]
+app.config["DISCORD_PUBLIC_KEY"] = os.environ["DISCORD_PUBLIC_KEY"]
+app.config["DISCORD_CLIENT_SECRET"] = os.environ["DISCORD_CLIENT_SECRET"]
+
+
+# Clear any existing global slash commands
+discord.update_slash_commands()
+
+
+# Simple command to mention a friend
+@discord.command(name="High Five", type=ApplicationCommandType.USER)
+def highFive(ctx):
+    return f"<@{ctx.author.id}> wants to say hello to <@{next(iter(ctx.members))}>"
```

Comment:
@Breq16 One more thing about the `Context` class, this field should'nt be accessible directly to avoid the use of `next(iter))` ? Because the member is in an object with his `id` as key, but since the `id` is in the object, we don't need it ? Or maybe in others cases there is several memebers ?